src/test/auto-completer_test.ts

```diff
@@ -474,6 +474,79 @@ suite('AutoCompleter', () => {
               'polymer/element-with-databinding.html', internalPropUsePosition),
           databindingCompletions);
     });
+
+
+    test(`autocomplete css custom properties`, async() => {
+      const {client} = await createTestEnvironment();
+      await client.openFile('index.html', `
+        <style>
+          div {
+            --foo: red;
+            --bar: var(--baz);
+            --fizz: {--floo: var(--flub);};
+          }
+        </style>
+      `);
+      const declaractionPosition = {line: 3, column: 16};
```

Comment:
Is this correct? In vscode this spot corresponds to the location right after `div`
![screen shot 2017-12-06 at 10 12 05 am](https://user-images.githubusercontent.com/6173664/33677440-a53e5a04-da6d-11e7-8f5d-0ae898c23a1d.png)

Reply:
Tools report 1-indexed, code uses 0-indexed